frontend/integration-tests/tests/kubevirt/vm.actions.scenario.ts

```diff
@@ -104,7 +104,7 @@ describe('Test VM actions', () => {
 
 describe('Test VM Migration', () => {
   const testVm = getVmManifest('Container', testName);
-  const vm = new VirtualMachine(testVm.metadata.name, testVm.metadata.namespace);
+  const vm = new VirtualMachine({...testVm.metadata});
```

Comment:
is the spread necessary here? is the `vm` or `testVm` going to mutate?

Reply:
Nope, that's a mistake, removed.